Jenkinsfile

```diff
@@ -38,16 +38,18 @@ RELEASE_SUFFIX=${RELEASE_SUFFIX}
 GIT_BRANCH=${env.BRANCH_NAME}
```

Comment:
The root Jenkinsfile should be made the same as the current master branch. It seems you have copied over the Jenkinsfile from your local master branch but that one was not up-to-date with the master branch on github. Here's how I suggest we fix it.

In the openvdc root directory:

```
# Bring your local master up-to-date
git checkout master
git pull --rebase

# Merge updated local master into your branch. This will cause a merge conflict on Jenkinsfile.
git checkout rpm_cleanup
git merge master

# Get the Jenkinsfile from current master on github.
curl https://raw.githubusercontent.com/axsh/openvdc/master/Jenkinsfile -o Jenkinfile

# Solve merge conflict.
git add Jenkinsfile
git commit
```